src/components/icon/icon.js

```diff
@@ -55,6 +60,116 @@ const IconType = {
  * @extends {Component}
  */
 class Icon extends Component {
+  defaultColor: string = style.white;
+  activeColor: string = style.brandColor;
+  className: string = '';
+
+  /**
+   * @constructor
+   * @param {Object} props - component props
+   */
+  constructor(props: Object) {
+    super(props);
+    if (props.path) {
+      const {id} = props;
+      this.className = `playkit-icon-${id}`;
+      // Avoid from override existing classes
+      if (!document.getElementsByClassName(this.className).length) {
+        this.createDynamicIconClass(props);
+      }
+    }
+  }
+
+  /**
+   * Creates a dynamic icon class with background image settings
+   *
+   * @param {Object} props - component props
+   * @returns {void}
+   * @memberof Icon
+   */
+  createDynamicIconClass = (props: Object) => {
+    const {path, state, color, activeColor} = props;
+    const fillColor = this.getFillColor(state, color, activeColor);
+    const pathTag = this.getPathTag(path, fillColor);
+    const svgUrl = this.getSVGUrl(pathTag);
+    const css = `.${this.className} { background-image: ${svgUrl}; }`;
+    const style = document.createElement('style');
+    style.type = 'text/css';
+    document.getElementsByTagName('head')[0].appendChild(style);
+    style.appendChild(document.createTextNode(css));
```

Comment:
can't you use inline style?

Reply:
No, we need to allow apps to override those classes if they're want to use other icons.